### PR TITLE
style: explicit values for flex shorthand

### PR DESCRIPTION
**Pull request by asangma**
**Related Issue:** #302

## Summary
Also includes
* box-sizing
* tweak for action > icon centering inside of `slot-container`

### CODE REVIEW

File: src/assets/styles/_base.scss [hunk 1]
```diff
@@ -1,9 +1,13 @@
 :host {
   background-color: var(--calcite-app-background);
+  box-sizing: border-box;
   color: var(--calcite-app-foreground);
   font-family: var(--calcite-app-font-family);
   font-size: var(--calcite-app-base-font-size);
   line-height: var(--calcite-app-line-height);
+  * {
+    box-sizing: border-box;
```
Comment (driskull) [inline src/assets/styles/_base.scss]: yusssss!

File: src/components/calcite-action/calcite-action.scss [hunk 2]
```diff
@@ -50,6 +50,10 @@
   }
 }
 
+.slot-container {
```
Comment (asangma) [inline src/components/calcite-action/calcite-action.scss]: @driskull This fixes #518

Comment (driskull) [reply src/components/calcite-action/calcite-action.scss]: awesome

File: src/assets/styles/_base.scss (hunk 1 above)
Comment (asangma) [reply src/assets/styles/_base.scss]: Yew!

File: src/components/calcite-action/calcite-action.scss (hunk 2 above)
Comment (asangma) [reply src/components/calcite-action/calcite-action.scss]: 🤘🏽

**Pull request merged**